### ISSUE TITLE
Script doesnt start on a S22 ultra

### ISSUE THREAD

**Issue by cgaming117** [Fate-Grand-Automata/FGA#1420]
### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)

### FGO server

NA

### FGA build number

1856

### Describe the bug

I recently upgraded to the 22 ultra and the app does not function when starting the script ingame. When i try to record it gives invalid media projection.

### Video

#
![Screenshot_20221029-005551_FateGO](https://user-images.githubusercontent.com/116938149/198816659-ca8fb848-3697-4a57-b294-c1062101f6fa.jpg)

### Device model

Samsung s22 ultra

### Android version

12

### Screen size

1440x3038

### RAM

12GB

**Comment by reconman** [comment]
Try stopping and starting the FGA service to trigger a new MediaProjection dialog.

If that doesn't help, you could try this build https://github.com/Fate-Grand-Automata/FGA/suites/9029438392/artifacts/416748651

It was originally made for Android 13, but you're still on 12, right?

**Comment by cgaming117** [comment]
Yes, 13 hasnt been released yet for this model as far as i know. Ill try that apk. Thanks.

**Comment by cgaming117** [comment]
Thank you that build worked fine and the recording worked on it too. I wonder what the issue was between them

**Comment by cgaming117** [comment]
Unfortunately im having a new issue now. 
![Screenshot_20221103-010304_FateGO](https://user-images.githubusercontent.com/116938149/199656729-a6011e43-af8d-43bf-80f1-271eaf325f11.jpg)

**Comment by reconman** [comment]
Is it battery optimization?
